devicetypes/smartthings/ikea-button.src/ikea-button.groovy

```diff
@@ -374,6 +384,20 @@ private Map getButtonEvent(Map descMap) {
 				buttonNumber = OPENCLOSESTOP_BUTTONS_ENDPOINTS[endpoint].STOP
 			}
 		}
+	} 
+	else if (isEWeLink()) {
+		if (descMap.clusterInt == zigbee.ONOFF_CLUSTER) {
+			buttonNumber = 1
+			if (descMap.commandInt == 0x00) {
+				buttonState = "held"
+			} 
+			else if (descMap.commandInt == 0x01) {
+				buttonState = "double"
+			}
+			else {
+				buttonState = "pushed"
+			}
```

Comment:
Please, could You reformat this section ?
`} else if {`

Reply:
I have modified the format, please recheck.